llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -1448,29 +1448,58 @@ class LoopVectorizationCostModel {
   /// Returns true if \p I is a memory instruction in an interleaved-group
   /// of memory accesses that can be vectorized with wide vector loads/stores
   /// and shuffles.
-  bool interleavedAccessCanBeWidened(Instruction *I, ElementCount VF);
+  bool interleavedAccessCanBeWidened(Instruction *I, ElementCount VF) const;
```

Comment:
Better make it in separate NFC patch

Reply:
Would be good to also include the additional DEBUG printing as part of the NFC patch

Reply:
https://github.com/llvm/llvm-project/pull/96681